<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PauseWidget</class>
 <widget class="QWidget" name="PauseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="midSpeedBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius:10px;
background-image:url(&quot;E://uiProject//midSpeed.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="slowSpeedbtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius:10px;
background-image:url(&quot;E://uiProject//slowSpeed.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="speedLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>快速</string>
   </property>
  </widget>
  <widget class="QLabel" name="curSpeedLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前速度:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fastSpeedBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius:10px;
background-image:url(&quot;E://uiProject//fastSpeed.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pausedLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius:10px;
background-image:url(&quot;E://uiProject//paused.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pauseBackground">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-90</y>
     <width>561</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/uiProject/pauseBackground1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>slowSpeedbtn</zorder>
  <zorder>speedLabel</zorder>
  <zorder>curSpeedLabel</zorder>
  <zorder>pausedLabel</zorder>
  <zorder>pauseBackground</zorder>
  <zorder>backBtn</zorder>
  <zorder>midSpeedBtn</zorder>
  <zorder>fastSpeedBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
